enhancements/sig-architecture/58-addon-placement-annotation/README.md

```diff
@@ -0,0 +1,295 @@
+# Setting Up Addon Configuration 
+
+## Release Signoff Checklist
+
+- [] Enhancement is `implemented`
+- [] Design details are appropriately documented from clear requirements
+- [] Test plan is defined
+- [] Graduation criteria for dev preview, tech preview, GA
+- [] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+The proposal introduces some ways to help user to set up configuration for their addons.
+
+## Motivation
+
+We want to provide a way to help user to set up configuration for their addons with current addon APIs
+
+## Use cases
+
+1. As a user, I have a configuration for my all managed cluster addons, the addons share the same configuration.
```

Comment:
I think we should have some details here.

Reply:
use the specific use case